backend/src/estark/json_exporter/mod.rs

```diff
@@ -65,12 +65,12 @@ pub fn export<T: FieldElement>(analyzed: &Analyzed<T>) -> PIL {
             }
             StatementIdentifier::PublicDeclaration(name) => {
                 let pub_def = &analyzed.public_declarations[name];
-                let pub_ref = &pub_def.polynomial;
-                let poly_id = pub_ref.poly_id.unwrap();
+                let symbol = &analyzed.definitions[&pub_def.polynomial.name].0;
+                assert_eq!(symbol.kind, SymbolKind::Poly(PolynomialType::Committed));
```

Comment:
In practice I agree but in theory does a public value have to expose a committed polynomial? Fixed polynomials seem fine too. Or do we have a cleaner error constraining this somewhere else?